# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/scipy/hotfix-meson.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- scipy-1.18.0/meson.options.vanilla	2026-08-19 19:03:13.521737532 +0300
+++ scipy-1.18.0/meson.options	2026-08-19 19:03:28.273789151 +0300
@@ -26,7 +26,7 @@
                      'to either pure Python code or Cython code, depending on ' +
                      'the implementation).')
 option('use-system-libraries', type: 'array',
-        choices : ['none', 'all', 'auto', 'boost.math', 'qhull'], value : ['none'],
+        choices : ['none', 'all', 'auto', 'boost.math', 'qhull'], value : ['all'],
         description: 'Choose which system libraries for subprojects ' +
                      'if they are available.')
 option('_without-fortran', type: 'boolean', value: false,

